app/controllers/app/task_controller.rb

```diff
@@ -0,0 +1,7 @@
+module App
+  class TaskController < App::ApplicationController
+    def index
+      @tasks = TaskService.new.task_list params:
+    end
+  end
+end
```

Comment:
The same comment here.
```suggestion
class TaskController < App::ApplicationController
  def index
    @tasks = TaskService.new.task_list params:
  end
end
```